timesketch/lib/analyzers/contrib/misp_analyzer_test.py

```diff
@@ -0,0 +1,89 @@
+"""Test for MISP"""
```

Comment:
Same comment as for the hashlookup_analyzer_test; you're not testing the analyzer, but the code that you write in this file :)

Reply:
Fixed in c0fd536f2fea435e66ecd4d5cf919714474952e8.